### PR TITLE
 Code review

### PR DESCRIPTION
**Pull request by edubew**
## In this activity I did the following:
- Created a new repository edubew/code-review
- Copied index.html code from the microverse-trials project.
- Copied style.css code from the microverse-trials project.
- Created a new branch review-code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <!DOCTYPE html>
-<html>
-    <meta charset="utf-8">
```
Comment (edubew) [inline index.html:3]: Please remove this section of the code

File: index.html [hunk 2]
```diff
@@ -15,13 +18,13 @@
             </nav>
         </header>
         <section>
-            <div>
```
Comment (edubew) [inline index.html:18]: Please add a class or id to your div tags to make sure that anther person going through your code can understand what you are doing

**Review by edubew — COMMENTED** [review]
## Highligts
- Check and fix linters errors

## Required changes:

_check the comments under the review_

**Review by Wes-Isaac — APPROVED** [review]
Hi @edubew, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**